apps/frontend-client/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Build environment
+FROM node:17-alpine as builder
```

Comment:
Är det någon anledning att vi kör version 17? Annars tycker jag vi ska försöka få upp den till en senare version typ 21-alpine

Reply:
Roger, totally agree! Wasn't aware of later versions of node:alpine, I'll make that change before merging. 

Please use English as working language, as this is the language we strive to use for communication across the organization. I'll be of importance as we bring on international contributors to our project!